### DISCUSSION TITLE
Raspberry has internet access but not clients

### DISCUSSION THREAD

**Discussion by janerik** [RaspAP/raspap-webgui#967] [Q&A]
Hey,

I have a problem wit some inconsistencies. 
I've setup the Raspap to connect with wifi0 to a network and provide this via wifi1 (external wifi antenna) or eth0. I've setup wifi1 and eth0 with two distinct dhcp regions. 
Everything worked the moment I configured it. I powered down the raspberry and wanted to test it again today before going on vacation. But somehow it doesn't work anymore.
The connection to the network with wifi0 works, if I ssh into the pi I can ping google and update etc. But if I want to connect to a site with my browser it doesn't work. Also wifi0 doesn't appear in the networking tab of Raspap.

Can anyone help me with this. What is it that I am holding wrong here?

**Comment by billz** [comment]
For troubleshooting advice start here:
https://docs.raspap.com/ap-basics/#troubleshooting
